core/src/main/java/org/mapstruct/MapperConfig.java

```diff
@@ -329,5 +329,14 @@ MappingInheritanceStrategy mappingInheritanceStrategy()
      */
     Class<? extends Exception> unexpectedValueMappingException() default IllegalArgumentException.class;
 
+    /**
+     * Flag indicating whether the creation of a time stamp in the {@code @Generated} annotation should be suppressed.
```

Comment:
creation = addtion?

Reply:
Mention somehow this overrides the processor setting (in the pom.xml in case of maven)

Reply:
Good point. I'll change it and mention the override of the processor setting.